functions/nodejs-apps.bash

```diff
@@ -202,3 +202,142 @@ nodered_setup() {
     dashboard_add_tile "nodered"
   fi
 }
+
+## Function for downloading zigbee2mqtt to current system
+##
+##    zigbee2mqtt_download(String prefix)
+##
+zigbee2mqtt_download() {
+  echo -n "$(timestamp) [openHABian] Downloading zigbee2mqtt... "
+  if ! cond_redirect mkdir /opt/zigbee2mqtt; then echo "FAILED (mkdir /opt/zigbee2mqtt)"; fi
+  if ! cond_redirect chown openhabian /opt/zigbee2mqtt; then echo "FAILED (chown /opt/zigbee2mqtt)"; fi
+  if ! cond_redirect chgrp openhab /opt/zigbee2mqtt; then echo "FAILED (chgrp /opt/zigbee2mqtt)"; fi
+  if ! cond_redirect sudo -u "${username:-openhabian}" git clone https://github.com/Koenkk/zigbee2mqtt.git "/opt/zigbee2mqtt"; then echo "FAILED (git clone)"; return 1; fi
+}
```

Comment:
Make sure to use consistent capitalization of Zigbee2MQTT in this function as well.

Reply:
done